### DISCUSSION TITLE
error when installing brew

### DISCUSSION THREAD

**Discussion by umutkavakli** [Homebrew/discussions#3135] [Linux]
Trying to install brew on ubuntu 20.04
using `/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"`
![WhatsApp Image 2022-04-03 at 15 07 13](https://user-images.githubusercontent.com/69873227/161427246-7bb3499d-d7c6-4dab-a273-71826034d7c1.jpeg)
I searched some discussions about this topic but I couldn't find any solution.

**Comment by umutkavakli** [comment]
Also, when I check the ruby version with `ruby -v`
I got this version: `ruby 2.7.0p0 (2019-12-25 revision 647ee6f091) [x86_64-linux-gnu]`

**Reply by SMillerDev** [reply]
You'll have to install ruby 2.6 first.

**Comment by carlocab** [comment]
Please [don't use screenshots](https://meta.stackoverflow.com/a/285557) when sharing the output of commands in your terminal. They're often difficult to read and impossible to reuse (e.g. when looking up error messages). Instead, copy the text here and wrap them in three back-ticks (```) on the line before and on the line after to format them properly.

**Comment by gromgit** [comment]
What @carlocab said, please post text as text, not screenshots.

That said, it looks like either your Homebrew cache directory (`/home/ginger/.cache/Homebrew`) is not writable, or you already have previous broken portable-ruby downloads that have incorrect permissions. Probably the fastest way to fix this: `rm -fr /home/ginger/.cache/Homebrew`, then run the installer again.

**Reply by umutkavakli** [reply]
sorry for the screenshot, i tried what you said but i am still getting same error messages.
